hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java

```diff
@@ -120,8 +122,24 @@ protected void init680() {
 			.online(false)
 			.withColumns("TAG_TYPE", "TAG_CODE", "TAG_SYSTEM", "TAG_ID", "TAG_VERSION", "TAG_USER_SELECTED");
 
+		version.onTable("HFJ_RES_VER_PROV")
+			.addIndex("20230510.1", "IDX_RESVERPROV_RES_VER_PID")
+			.unique(false)
+			.withColumns("RES_VER_PID")
+			.failureAllowed();
+		version.onTable("HFJ_RES_VER_PROV")
+			.addIndex("20230510.2", "IDX_RESVERPROV_RES_PID")
+			.unique(false)
+			.withColumns("RES_VER_PID");
```

Comment:
confused: these look identical: same columns on same table.  Did you mean?
```suggestion
		version.onTable("HFJ_RES_VER_PROV")
			.addIndex("20230510.2", "IDX_RESVERPROV_RES_PID")
			.unique(false)
			.withColumns("RES_PID");
```